app/test_main.py

```
@@ -1,3 +1,25 @@
 from app.main import get_human_age
+import pytest
+from typing import List
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+
+    ],
+)
+def test_get_human_age(
+        cat_age: int,
+        dog_age: int,
+        expected: List[int]
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
```

This violates checklist item 3.5: 'Tests should check incorrect data types handling: Non-integer inputs should raise the correct exception (e.g., TypeError) if such behavior is implemented/specified. (Define in tests based on expected behavior.)'

Please add tests for non-integer inputs (e.g., strings, floats, None) for cat_age and dog_age.